### PR TITLE
[Xedra Evolved] Bugfix add Jotunn corpse

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
Adds Exodii Jotunn corpse

#### Describe the solution
Adds corpse item

#### Describe alternatives you've considered
None

### CODE REVIEW

File: data/mods/Xedra_Evolved/items/corpses.json [hunk 1]
```diff
@@ -37,5 +37,21 @@
...
     "flags": [ "TRADER_AVOID", "NO_REPAIR" ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "broken_exodii_quad_jotunn",
+    "symbol": ",",
+    "color": "light_gray",
+    "name": { "str": "broken Exodii quadruped" },
+    "category": "other",
+    "description": "A broken Exodii Jotunn walker.  Still looks incredibly intimidating despite being permanently inoperative, due to the sheer size, mass, and protruding weapons.  Could be gutted for parts.",
```
Comment (GuardianDll) [inline data/mods/Xedra_Evolved/items/corpses.json:48]: > Could be gutted for parts
 Should add a disassembly recipe then?

Comment (Maleclypse) [reply data/mods/Xedra_Evolved/items/corpses.json:48]: Has a harvest like the normal exodii quad, so if you dissect it you can get cbms.   I'll check and see what all the broken_quad corpse ties into though and make sure I'm not just adding another error :)

Comment (Maleclypse) [reply data/mods/Xedra_Evolved/items/corpses.json:48]: You were correct, it also needed a deconstruct. Thanks!

File: data/mods/Xedra_Evolved/recipes/recipes_deconstruction.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      [ [ "mil_plate", 2 ] ],
+      [ [ "hard_steel_armor", 10 ] ],
+      [ [ "sheet_metal_small", 40 ] ],
+      [ [ "motor_small", 1 ] ],
+      [ [ "cable", 10 ] ],
+      [ [ "omnicamera", 1 ] ],
+      [ [ "betavoltaic", 20 ] ]
+    ]
+  }
+]
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/recipes/recipes_deconstruction.json:52]:
```suggestion
]

```

**Pull request merged**